tests/Doctrine/Tests/ORM/Functional/MergeVersionedOneToManyTest.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace Doctrine\Tests\ORM\Functional;
+
+use Doctrine\ORM\OptimisticLockException;
+use Doctrine\ORM\ORMException;
+use Doctrine\Tests\Models\VersionedOneToMany\Article;
+use Doctrine\Tests\Models\VersionedOneToMany\Category;
+
+/**
+ *
+ * @group MergeVersionedOneToMany
+ */
+class MergeVersionedOneToManyTest extends \Doctrine\Tests\OrmFunctionalTestCase
+{
+    protected function setUp()
+    {
+        parent::setUp();
+
+        try {
+            $this->_schemaTool->createSchema(
+                [
+                    $this->_em->getClassMetadata('Doctrine\Tests\Models\VersionedOneToMany\Category'),
+                    $this->_em->getClassMetadata('Doctrine\Tests\Models\VersionedOneToMany\Article'),
+                ]
+            );
+        } catch (ORMException $e) {
+        }
+    }
+
+    /**
+     * This test case tests that a versionable entity, that has a oneToOne relationship as it's id can be created
+     *  without this bug fix (DDC-3318), you could not do this
```

Comment:
All white-spaces & lines are fixed. And I modified the comment as it is here just by a copy paste.